Lab6.ipynb

```diff
@@ -0,0 +1,443 @@
+      "text/plain": [
+       "<Figure size 1296x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "a1 = stats.poisson.rvs(100, size = 10000)\n",
+    "b1 = np.array([])\n",
+    "signal = 250\n",
+    "for i in a1:\n",
+    "    b1 = np.append(b1, i + signal)\n",
+    "    \n",
+    "plt.hist(b1, bins = 200)\n",
+    "plt.title('Signal')\n",
+    "plt.xlabel('Measurement')\n",
+    "plt.ylabel('Counts')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see a Poisson distribution with a mean of 350, which is what we had before."
```

Comment:
Explanation Problem 3B:
- Yep, we are supposed to see this distribution, so this part makes sense here.